rules/gsuite_activityevent_rules/gsuite_workspace_calendar_external_sharing.py

```diff
@@ -0,0 +1,25 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    if not all(
```

Comment:
I thought the first commit was logically equivalent to:
```suggestion
    if event.get('name', '') == 'CHANGE_CALENDAR_SETTINGS'
    and deep_get(event, 'parameters','SETTING_NAME', default = '') != 'SHARING_OUTSIDE_DOMAIN'
```

Reply:
as a positive assertion it would read: 
```python
if event.get("name", "") == "CHANGE_CALENDAR_SETTING" and  deep_get(event, "parameters", "SETTING_NAME", default="") == "SHARING_OUTSIDE_DOMAIN":
    pass
else:
    return False
```

Reply:
I think the not all is fine